### ISSUE TITLE
input for imageColorSyncProfileData

### ISSUE THREAD

**Issue by typemytype** [typemytype/drawbot#344]
the input for the `imageColorSyncProfileData` could also be a `.icc` file next to an NSData object.

Creating a NSData object does feel very drawBotisch or pythonesk. 

Creating a NSData object on the fly seems hard... see the spec for an .icc file http://www.color.org/specification/ICC1v43_2010-12.pdf

[example .icc file](https://github.com/haasn/cms/blob/master/Adobe%20ICC%20Profiles%20(end-user)/RGB/AppleRGB.icc)

```python
import AppKit

fill(1, 0, 0)
rect(20, 20, width()-40, height()-40)

data = AppKit.NSData.dataWithContentsOfFile_('path/to/AppleRGB.icc')

saveImage("test.tiff", imageColorSyncProfileData=data)
```

so a proposal: `saveImage("test.tiff", imageColorSyncProfileData='path/to/AppleRGB.icc')`

discussion started [here](https://forum.drawbot.com/topic/226/saving-image-with-different-colorprofile)

**Comment by justvanrossum** [comment]
I agree nsdata is clumsy, but can a color profile influence the number of output color channels?

**Comment by typemytype** [comment]
nsdata is indd clumsy ;)
